tasks/package.js

```diff
@@ -117,25 +192,31 @@ module.exports = function(grunt) {
       tasks.push('composer:drupal-scaffold');
     }
 
-    if (this.args[0] && this.args[0] === 'compress') {
+    if (archive || (this.args[0] && this.args[0] === 'compress')) {
       grunt.loadNpmTasks('grunt-contrib-compress');
       grunt.config('compress.package', {
         options: {
           archive: destPath + '.tgz',
-          mode: 'tgz',
-          gruntLogHeader: false
+          mode: 'tgz'
         },
         files: [
           {
             expand: true,
             dot: true,
-            cwd: grunt.config.get('config.buildPaths.packages') + '/' + packageName,
+            cwd: destPath,
             src: ['**']
           }
         ]
       });
 
       tasks.push('compress:package');
+
+      if (destPath !== finalPath) {
+        grunt.config('shell.mvArchive', {
+          command: 'mv ' + destPath + '.tgz ' + finalPath + '.tgz'
```

Comment:
Hm, maybe that restriction is only outside the container. mv is an order of magnitude faster than copy, so if it works I think we should keep it until we have reason to switch to copy.

A bigger issue is windows compat. For portability we should just switch to an fs.rename[Sync].